### ISSUE TITLE
Feature Request: set all channels config for same channel relay

### ISSUE THREAD

**Issue by retiform** [42wim/matterbridge#228]
Be great to auto relay **all**  existing channels with the same name, instead of adding all channel names to the auto relay manually.

**Comment by 42wim** [comment]
You mean autodetect all available channels and relaying them all ?

**Comment by retiform** [comment]
Yes

**Comment by Madji** [comment]
private messages? possible or not

**Comment by 42wim** [comment]
Not possible, maybe for matterbridge 2.0

**Comment by 42wim** [comment]
retiform: If enough people want this, I'll think about implementing it. Also PR are welcome of course.

**Comment by eburghar** [comment]
agree. it would be great. even with the possibility of auto creating channel if the user has the right to do.

:+1:
